extensions/sql/policy/store/README.md

```diff
@@ -0,0 +1,39 @@
+# SQL Policy Store
+
+Provides SQL persistence for policies.
+
+## Prerequisites
+
+Please apply this [schema](docs/schema.sql) to your SQL database.
```

Comment:
The link is incorrect, should be `../docs/schema.sql`. Also, that schema is not valid for Postgres, please adapt accordingly (remove size constraint, as commented before) and add comment "this is only tested for postgres" etc. for comparison check [here](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/blob/3b7d17938562948e34522d4c919e725269ad0b8b/extensions/sql/transfer-process-store/doc/ddl_postgres.sql)